charts/alfresco-search-service/templates/ingress.yaml

```
@@ -24,6 +23,9 @@ spec:
     {{- end }}
   {{- end }}
   rules:
+  {{- if .Values.ingress.hostName }}
+  - host: {{ tpl .Values.ingress.hostName . }}
+  {{- else }}
```

This `else` doesn't look to have a matching closing `end`, but I would even argue that one may want to specify both the hostname to bind the ingress on together with a `HTTPIngressRuleValue`

```suggestion
  {{- end }}
```

---

I'll fix the else to end like suggested

---

Not a big deal, that's just the name of the object used in the rule: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#httpingressrulevalue-v1-networking-k8s-io